src/include/block.h

```diff
@@ -211,6 +211,13 @@ struct __wt_bm {
     size_t maplen;
     void *mapped_cookie;
 
+    /* Tiered storage */
```

Comment:
Nit: Layering wise would it make sense to fully generalize this to "Is multi table", or similar, with a comment noting that the user of multi-table block manager is tiered storage?

Specifically updating this comment and changing `is_tiered` to `is_multi_handle` (or a name of your choosing)

Reply:
Good point. I've updated the variable name as you suggested.